### ISSUE TITLE
Support prosody internal_hashed authentication method

### ISSUE THREAD

**Issue by nathan-sain** [luvolondon/fvtt-module-jitsiwebrtc#55]
In my experience, the module does not work with Jitsi Meet when the Prosody Virtual Host for the Jitsi hosts domain is configured to use the internal_hashed method for storing passwords. The Jitsi documentation for securing a domain specifies internal_hashed as the authentication method https://jitsi.github.io/handbook/docs/devops-guide/secure-domain.

If my assumption is correct and it is not possible to support the internal_hashed authentication method it would help to update tutorials and the README.md document for this repository to indicate that the internal_hashed authentication method is not supported.

Thank you for your consideration and for the time and effort spent on this module. It has helped make Foundry VTT the best option for my RP sessions.

**Comment by bekriebel** [comment]
It works with internal_hashed passwords. That's actually what I use on my own server. One thing to note is that the username is in the form of `username@yourdomain.tld` and not just `username`. I've been meaning to add a check to that to warn when it isn't entered in that format.

If doing that doesn't work, can you provide log output with any errors after turning on the debugging option?

**Comment by bekriebel** [comment]
I haven't heard back, but I don't believe this is an issue. Feel free to re-open if you are still having trouble.

**Comment by nathan-sain** [comment]
Just wanted to follow-up and verify that after further testing I was able to use the internal_hashed password store. Not sure what the problem was. I'm using the Docker build of Jitsi Meet and it's hard to tweak the config files. I was using the full user jid in the Foundry settings and was was seeing auth errors in the developer console, but at the same time was able to authenticate on the Jitsi meet web server. I started over and rebuilt the docker containers and now it works fine. 

Thanks so much for your time.
